.github/workflows/build.yml

```diff
@@ -45,8 +45,7 @@ jobs:
         java: [8, 8.0.192, 11.0.x, 11.0.3, 12, 13, 15, 16, 17 ]
     name: Java ${{ matrix.java }}
     env:
-      OS: ${{ matrix.os }}
-      JAVA: ${{ matrix.java-version }}
+      JAVA: ${{ matrix.java }}
```

Comment:
Do you know how this is currently working? I'm confused that there's no `java.version` set on the matrix configuration, only `java` :|

Reply:
Actually I don't know, it's a bit weird I admit...